TYPO3.Neos/Resources/Public/JavaScript/Shared/Utility.js

```diff
@@ -13,6 +13,9 @@ define(function() {
 		 * @return {boolean}
 		 */
 		isValidJsonString: function(jsonString) {
+			if(typeof jsonString !== 'string') {
```

Comment:
nope, haven't really done much of that since all that stuff is happening in the rewrite